packages/suite/src/views/wallet/coinmarket/p2p/offers/List/Quote.tsx

```diff
@@ -0,0 +1,379 @@
+import React from 'react';
+import { FormattedNumber } from 'react-intl';
+import styled from 'styled-components';
+import BigNumber from 'bignumber.js';
+import { useFormatters } from '@suite-common/formatters';
+import { Translation } from '@suite-components';
+import { Button, H3, Icon, Tooltip, useTheme, variables } from '@trezor/components';
+import regional from '@wallet-constants/coinmarket/regional';
+import { useCoinmarketP2pOffersContext } from '@wallet-hooks/useCoinmarketP2pOffers';
+import { CoinmarketProviderInfo } from '@wallet-components';
+import { P2pQuote, P2pQuotesRequest } from 'invity-api';
+import { Avatar } from '../Avatar';
+
+const Wrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    border-radius: 8px;
+    flex: 1;
+    width: 100%;
+    min-height: 150px;
+    margin-bottom: 20px;
+    padding-bottom: 16px;
+    background: ${({ theme }) => theme.BG_WHITE};
+`;
+
+const Main = styled.div`
+    display: flex;
+    margin: 30px 30px 0 30px;
+    justify-content: space-between;
+    padding-bottom: 20px;
+    border-bottom: 1px solid ${({ theme }) => theme.STROKE_GREY};
+
+    @media (max-width: ${variables.SCREEN_SIZE.SM}) {
+        flex-direction: column;
+        margin: 20px 20px 0 20px;
+    }
+`;
+
+const MainColumn = styled.div`
+    display: flex;
+    flex: initial;
+    flex-direction: column;
+    width: 33.33%;
+
+    @media (max-width: ${variables.SCREEN_SIZE.SM}) {
+        width: auto;
+    }
+`;
+
+const MainColumnRight = styled.div`
+    display: flex;
+    flex: initial;
+    width: 33.33%;
+    justify-content: flex-end;
+
+    @media (max-width: ${variables.SCREEN_SIZE.SM}) {
+        justify-content: center;
+        padding-top: 10px;
+    }
+`;
+
+const MainHeading = styled.div`
```

Comment:
It could use some bottom margin
![image](https://user-images.githubusercontent.com/3729633/193290687-f9366ad2-9234-4bb2-a2fe-5a1c8940aa80.png)

Reply:
Better like this?
<img width="555" alt="image" src="https://user-images.githubusercontent.com/37251117/193544580-02c5e08e-c1ed-4ad2-8b95-9fd522d43fc9.png">